user-interface/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "main": "Gruntfile.js",
+  "devDependencies": {
+    "grunt-contrib-qunit": "latest",
+    "grunt": "latest"
+  }
+}
```

Comment:
The JavaScript tests should also be executable if they are declared in the modules.